std_msgs/Header header
float32 horizontal_fov
float32 vertical_fov
float32 last_gradient
float32 average_gradient
float32 altitude
int64 idb
bool acceptable